src/components/projects/projectId/heuristics/HeuristicsHeader.tsx

```diff
@@ -160,84 +163,101 @@ export default function HeuristicsHeader(props: HeuristicsHeaderProps) {
                         disabledOptions={[false, false]}
                         selectedOption={(option: string) => executeOption(option)} buttonClasses={`${style.actionsHeight} text-xs whitespace-nowrap`} dropdownClasses="mr-3" dropdownItemsWidth='w-40' dropdownWidth='w-32'
                         iconsArray={['IconCode', 'IconBolt']} useFillForIcons={[false, true]} />
-                </Tooltip>) : (<Tooltip content={TOOLTIPS_DICT.HEURISTICS.DISABLED_NEW_HEURISTIC} color="invert">
-                    <button type="button" disabled={true}
-                        className="mr-3 inline-flex items-center justify-center w-full rounded-md border border-gray-300 shadow-sm px-4 py-1.5 bg-white text-xs font-medium text-gray-700 opacity-50 cursor-not-allowed focus:ring-offset-2 focus:ring-offset-gray-400"
-                        id="menu-button" aria-expanded="true" aria-haspopup="true">
-                        New heuristic
-                        <svg className="-mr-1 ml-2 h-5 w-5" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"
-                            fill="currentColor" aria-hidden="true">
-                            <path fillRule="evenodd"
-                                d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
-                                clipRule="evenodd" />
-                        </svg>
-                    </button>
-                </Tooltip>)}
+                </Tooltip>) : (
+                    <KernButton
+                        onClick={() => { }}
```

Comment:
empty function not needed imo (not sure if KernButton can handle it though but since disabled it should be fine 🤔